src/documentation/WizardHookForm/FormOne.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import TextInput from '../../components/TextInput';
+import TextArea from '../../components/TextArea';
+import Blockquote from '../../components/Blockquote';
+
+function FormOne(props) {
+
+    return (
+        <>
+            <div style={{marginBottom:'2rem'}}>
+              <Blockquote title="Regular Blockquote">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+                nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+                sed diam voluptua.
+              </Blockquote>
+              </div>
+              <TextInput
+                id="firstName"
+                name="firstName"
+                helperText="Enter your full first name"
+                labelText="First Name"
+                inputRef={props.register}
+              />
+                {/* <input
+                name="firstName"
+                ref={register}
+                /> */}
+              <TextInput
+                id="lastName"
```

Comment:
It might not be necessary to set the `id` prop anymore.

Reply:
sure, will add it